src/network/taraxa_capability.hpp

```diff
@@ -43,6 +43,13 @@ enum SubprotocolPacketType : ::byte {
   PacketCount
 };
 
+enum GetBlocksPacketRequestType : ::byte {
+
+  MissingHashes = 0x0,
+  KnownHashes,
+  RequestCount
```

Comment:
what is this count good for ?

Reply:
for the future use

Reply:
but I can remove it

Reply:
not really sure what future use that might be, but I might not see something. Either keep it but handle possible values of it correctly in other places or remove it....